### ISSUE TITLE
Where is the aws-amplify Docker file hosted?

### ISSUE THREAD

**Issue by bvirlet** [aws-amplify/amplify-hosting#3341]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-hosting/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-hosting/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting questions](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have removed any sensitive information from my code snippets and submission.

### App Id

d2zlsiaic5yx2j

### AWS Region

us-east-1

### Amplify Hosting feature

Deployments

### Question

Hello,

I'm trying to create a custom Docker image for AWS Amplify. Rather than rebuilding the image from scratch, I would like to base myself on the existing amplify-console/buildimage:latest. However, I can't find where this image is hosted.

In other words, I would like to do the following:

```
FROM amplify-console/buildimage:latest

…
```

Currently, `docker build` fails as follow:

```
 > [internal] load metadata for docker.io/amplify-console/buildimage:latest:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```

Thanks!

**Comment by hloriana** [comment]
Hi @bvirlet thanks for raising this question. I'm currently investigating what the proper location is for the image. In the mean time, here is the dockerfile that we use: https://github.com/aws-amplify/amplify-hosting/blob/main/images/latest/Dockerfile

**Comment by emendoza2** [comment]
Hi. Could I follow up with this issue? I am having similar trouble getting the docker image to work locally for testing.
